### PR TITLE
Fixing critical issues related to sonar code analysis in commons module.

### PR DESCRIPTION
**Pull request by nagachandra-immadisetty**

### CODE REVIEW

File: commons/src/main/java/io/hyscale/commons/config/SetupConfig.java [hunk 1]
```diff
@@ -36,23 +36,21 @@
...
 
 
-    private static final String generatedFilesDir = "generated-files";
-    private static final String logDir = "logs";
-    private static final String appsDirectory = "apps";
-    private static final String hyscale = "hyscale";
+    private static final String GENERATED_FILES_DIR = "generated-files";
+    private static final String LOGS_DIR = "logs";
+    private static final String APPS_DIR = "apps";
+    private static final String HYSCALE = "hyscale";
```
Comment (SameeraGrandhi) [inline commons/src/main/java/io/hyscale/commons/config/SetupConfig.java:42]: HYSCALE_DIR

File: commons/src/main/java/io/hyscale/commons/utils/NormalizationUtil.java [hunk 2]
```diff
@@ -42,7 +42,7 @@ public static String normalize(String name) {
     }
     
     public static String normalizeImageName(String imageName) {
-        imageName = imageName.replaceAll(" ", "");
+        imageName = imageName.replace(" ", "");
```
Comment (SameeraGrandhi) [inline commons/src/main/java/io/hyscale/commons/utils/NormalizationUtil.java:45]: Why was it not replaceAll ?

File: commons/src/main/java/io/hyscale/commons/utils/WindowsUtil.java [hunk 3]
```diff
@@ -58,7 +58,7 @@ public static String updateToHostFileSeparator(String filepath) {
 			return filepath;
 		}
 		if (isHostWindows()) {
-		    return filepath.replaceAll(ToolConstants.LINUX_FILE_SEPARATOR, WINDOWS_FS_MATCHER);
+		    return filepath.replace(ToolConstants.LINUX_FILE_SEPARATOR, WINDOWS_FS_MATCHER);
```
Comment (SameeraGrandhi) [inline commons/src/main/java/io/hyscale/commons/utils/WindowsUtil.java:61]: Why not replaceAll ?

File: commons/src/main/java/io/hyscale/commons/commands/CommandExecutor.java [hunk 4]
```diff
@@ -90,7 +90,7 @@ public static boolean execute(String command) {
             logger.error("Failed while executing command, error {}", ex.toString());
             return false;
         }
-        return commandResult == null || commandResult.getExitCode() != 0 ? false : true;
+        return commandResult.getExitCode() != 0 ? false : true;
```
Comment (SameeraGrandhi) [inline commons/src/main/java/io/hyscale/commons/commands/CommandExecutor.java:93]: Why is the null check removed ?

Comment (nagachandra-immadisetty) [reply commons/src/main/java/io/hyscale/commons/commands/CommandExecutor.java:93]: in _execute() method commandResult is being initialized with new keyword and this is never reassigned. So there is no possibility of commandResult is null. So I removed null checks as per sonar suggestions.

**Pull request merged**